### ISSUE TITLE
Babel Deprecation for Path#remove

### ISSUE THREAD

**Issue by rwjblue** [pangratz/ember-cli-htmlbars-inline-precompile#11]
```
Trace: Path#remove has been renamed to Path#dangerouslyRemove, removing a node is extremely dangerous so please refrain using it.
    at NodePath.remove (/Users/rjackson/Source/javascript/ember-legacy-views/node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/
babel-core/lib/babel/traversal/path/removal.js:29:11)
    at NodePath.babel.Transformer.ImportDeclaration (/Users/rjackson/Source/javascript/ember-legacy-views/node_modules/ember-cli-htmlbars-inline-precompile/no
de_modules/babel-plugin-htmlbars-inline-precompile/index.js:18:16)
    at NodePath.call (/Users/rjackson/Source/javascript/ember-legacy-views/node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/ba
bel-core/lib/babel/traversal/path/context.js:122:26)
    at NodePath.visit (/Users/rjackson/Source/javascript/ember-legacy-views/node_modules/ember-cli-babel/node_modules/broccoli-babel-transpiler/node_modules/b
abel-core/lib/babel/traversal/path/context.js:146:8)
    at TraversalContext.visitMultiple (/Users/rjackson/Source/javascript/ember-legacy-views/node_modules/ember-cli-babel/node_modules/broccoli-babel-transpile
```

**Comment by pangratz** [comment]
Thanks for reporting! This should have been fixed via #10 which is released in `0.1.1`. Can you check which version you are using and close this issue if upgrading `ember-cli-htmlbars-inline-precompile` removes warning?

**Comment by pangratz** [comment]
I'm going to close this as I think that this should be resolved. Please reopen if the problem still exists ...
